lms/templates/static_htmlbook.html

```diff
@@ -126,7 +126,7 @@
               <ul id="booknav" class="treeview-booknav">
                 <%def name="print_entry(entry, index_value)">
                   <li id="htmlchapter-${index_value}">
-                    <a class="chapter">
+                    <a href="#bookpage" class="chapter">
```

Comment:
This link should send focus to the wrapper with the `id` of `bookpage`, which contains the PDF file that's loaded.